docs/DeveloperGuide.md

```diff
@@ -471,19 +504,19 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Deleting a module
 
-1. Deleting a person while all persons are being shown
+1. Deleting a module.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+   1. Prerequisites: Multiple modules in the 'Current Modules'. e.g. CS2103T in 'Current Modules'.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `delete CS2103T`<br>
+      Expected: CS2103T module is deleted from 'Current Modules'. Details of the deleted module shown in the status message. Timestamp in the status bar is updated.
```

Comment:
What does the timestamp in the status bar refer to ah?

Reply:
I think it has got to do with the JavaFX status bar but I don't see it in the GUI

Reply:
Okay I'll remove it since its not needed.